CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
+- Command Generate - Generate MongoDB belongsTo.
```

Comment:
Looks very good with a 2 min review, and with tests 🤩
(Looks like a feature to me, not a fix)

Reply:
❤️